ignite/pkg/cosmosbuf/buf.go

```diff
@@ -177,16 +247,44 @@ func findSDKProtoPath(protoDir string) (string, error) {
 	return fmt.Sprintf("%s@%s/proto", paths[0], version), nil
 }
 
-// prepareSDK copy the cosmos sdk proto folder to a temporary directory
-// so we can skip the buf workspace.
-func prepareSDK(protoDir string) (string, error) {
+// CopySDKProtoDir copies Cosmos SDK proto folder to a temporary directory.
+// The temporary directory must be removed by the caller.
+func CopySDKProtoDir(protoDir string) (string, error) {
```

Comment:
we don't need to export this method

```suggestion
func copySDKProtoDir(protoDir string) (string, error) {
```

Reply:
fixed